diff --git a/belladonna/src/apis/api20100401_incoming_phone_number_api.rs b/belladonna/src/apis/api20100401_incoming_phone_number_api.rs
index a1137000..f91c89b8 100644
--- a/belladonna/src/apis/api20100401_incoming_phone_number_api.rs
+++ b/belladonna/src/apis/api20100401_incoming_phone_number_api.rs
@@ -483,7 +483,7 @@ pub async fn update_incoming_phone_number(
     let uri_str = format!(
         "{}/2010-04-01/Accounts/{AccountSid}/IncomingPhoneNumbers/{Sid}.json",
         configuration.base_path,
-        AccountSid = crate::apis::urlencode(params.account_sid),
+        AccountSid = crate::apis::urlencode(&params.account_sid),
         Sid = crate::apis::urlencode(params.sid)
     );
     let mut req_builder = configuration
@@ -497,9 +497,9 @@ pub async fn update_incoming_phone_number(
         req_builder = req_builder.basic_auth(auth_conf.0.to_owned(), auth_conf.1.to_owned());
     };
     let mut multipart_form_params = std::collections::HashMap::new();
-    if let Some(param_value) = params.account_sid {
-        multipart_form_params.insert("AccountSid", param_value.to_string());
-    }
+
+    multipart_form_params.insert("AccountSid", params.account_sid.clone());
+
     if let Some(param_value) = params.api_version {
         multipart_form_params.insert("ApiVersion", param_value.to_string());
     }
